source/code-snippets/usage-examples/findOne.ts

```diff
@@ -0,0 +1,40 @@
+import { FindOptions, MongoClient } from "mongodb";
+
+// Replace the uri string with your MongoDB deployment's connection string.
+const uri =
+  "mongodb+srv://<user>:<password>@<cluster-url>?writeConcern=majority";
+
+const client = new MongoClient(uri);
+
+type imdb = {
+  rating: number;
+  votes: number;
+  id: number;
+};
+
+interface MovieSchema extends Document {
+  title: string;
+  imdb: imdb;
+}
+
+async function run(): Promise<void> {
+  try {
+    await client.connect();
+
+    const database = client.db("sample_mflix");
+    const movies = database.collection<MovieSchema>("movies");
+    const query: object = { title: "The Room" };
```

Comment:
```suggestion
    const query = { title: "The Room" };
```

No need for the `object` annotation.